spec/models/match_spec.rb

```diff
@@ -43,6 +43,23 @@
     end
   end
 
+  context '#winner' do
+    before do
+      @undecided_match = create(:undecided_group_match)
+      @decided_match = create(:decided_playoff_match)
+    end
+
+    it 'returns a winner Team for a decided match' do
+      winner = @decided_match.winner
+      expect(winner).to be_a Team
```

Comment:
you should probably check if it's returning the correct team instead of any...

Reply:
Done